schema.ts

```diff
@@ -96,21 +93,26 @@ export enum CurrencyFormat {
   Financial = 'financial',
 }
 
-export interface CurrencySchema extends NumberSchema {
+export interface SimpleNumberSchema extends BaseNumberSchema {
+   // These need no other values for Coda's value formats.
```

Comment:
Not sure that's true? Date and Time have a `format` attribute, and DateTime has `dateFormat` and `timeFormat`.

And Percent needs to be associated with NumericSchema somehow (as well as ValueType.Number with no hint type).

Reply:
by this, you mean a schema like
```
{
  type: ValueType.Number;
  codaType: ValueHintType.Percent;
}
```

Is an okay format?

Reply:
Yeah, that's a valid format, and so is

```
{ type: ValueType.Number, codaType: ValueHintType.Percent, precision: 3}
```